docs/src/app/components/pages/components/RadioButton/ExampleSimple.jsx

```diff
@@ -30,6 +32,13 @@ const RadioButtonExampleSimple = () => (
         disabled={true}
         style={styles.radioButton}
       />
+      <RadioButton
+        value="ludicrous"
```

Comment:
I think that we can have one `RadioButton` in the first `RadioButtonGroup` with a custom `checkedIcon` and a unique value.
Then one disabled `RadioButton` in the second `RadioButtonGroup` that is selected by default.